documentation/user_docs/docs/manual/Manipulating_and_extracting_data_from_SQW_files_and_objects.rst

```diff
@@ -12,13 +12,13 @@ There are various different forms of input for this function, the purpose of whi
    my_cut = cut_sqw (data_source, proj, p1_bin, p2_bin, p3_bin)
 
 
-``data_source`` is a string giving the full filename (including path) of the input SQW file.
+Where ``data_source`` is a string giving the full filename (including path) of the input SQW file or just SQW object stored in memory.
```

Comment:
where ``data source`` is EITHER a string ..... SQW file or ELSE the VARIABLE NAME of an QW object stored in memory

Reply:
modified.